.github/workflows/helm_pr_deployment.yml

```
@@ -0,0 +1,93 @@
+name: Deploy PR to Kubernetes
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  deploy:
+    if: ${{ github.event.pull_request.head.repo.full_name == 'ls1intum/Artemis' }}
+    runs-on: ubuntu-latest
+    environment: k8s
```

Small suggestion (Will need a little adaption for the var):
```suggestion
    environment: 
        name: k8s
        url: https://pr${prNumber}.artemis-k8s.ase.cit.tum.de
```